<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackSelectDialog</class>
 <widget class="QDialog" name="TrackSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>631</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="chooseFilesGridLayout">
    <item row="0" column="3">
     <widget class="QPushButton" name="convertToTrack1PushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Convert from track 2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="chooseTrack2PushButton">
      <property name="text">
       <string>Choose</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="convertToTrack2PushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Convert from track 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="track1Label">
      <property name="text">
       <string>Track 1:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="chooseTrack1PushButton">
      <property name="text">
       <string>Choose</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="track2LineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="track2Label">
      <property name="text">
       <string>Track 2:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="track1LineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="trialsHorizontalLayout">
    <item>
     <widget class="QLabel" name="trialsLabel">
      <property name="text">
       <string>Trials: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="trialsLineEdit">
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="chooseSegmentHorizontalLayout">
    <item>
     <widget class="QLabel" name="chooseSegmentLabel1">
      <property name="text">
       <string>Use segment from </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="segmentStartTimestampLineEdit">
      <property name="inputMask">
       <string>99:99:99</string>
      </property>
      <property name="text">
       <string>::</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="chooseSegmentLabel2">
      <property name="text">
       <string>to</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="segmentStopTimestampLineEdit">
      <property name="inputMask">
       <string>99:99:99</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="trackLengthsAreNotEqualLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>60</red>
         <green>56</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Track lengths are not equal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TrackSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TrackSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
